### PR TITLE
Webpackify

### PR DESCRIPTION
**Pull request by i-like-robots**
Switch to using Webpack for all our client bundling needs. Uses latest n.Makefile so we can use ESLint 2 and JSX linting too.

Dev bundles compile in half the time, prod bundles have proper dead-code elimination (556kb vs. 624kb). So many wins!

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const config = {
```
Comment (matthew-andrews) [inline .eslintrc.js]: can be removed +gitignored now you updated this in `n-makefile`or not ?

Comment (i-like-robots) [inline .eslintrc.js]: Yes! done

**Comment by matthew-andrews** [comment]
looks good to me, bit worried about the failures though

**Comment by i-like-robots** [comment]
Failures look to be standard flakiness of this app but will double check the saucelabs tests.

**Pull request merged**